README.md

```diff
@@ -8,7 +8,7 @@ Requires Python 3.6
 
 Call the stix_shifter in the format of
 
-python stix_shifter.py `<data source>` `<input format>` `<STIX pattern>`
+python stix_shifter.py `<translator_module>` `<query or result>` `<data>`
```

Comment:
Ben's command line parser have some more changes in this section. Would that create any conflict?

Reply:
Most likely. He'll make the update in his PR though, as it stands with this commit this is how it will work